### ISSUE TITLE
Weather not working

### ISSUE THREAD

**Issue by haer0248** [marcopixel/SysDash#48]
Weather not working. It's display N/A
setting: **Asia,Taipei Taipei,Asia Asia, Taipei Taipei, Asia** still not working ;(

**Comment by efeurun** [comment]
```
[Rainmeter]
Group=SysDash | Weather
Update=10000
AccurateText=1
BackgroundMode=2
SolidColor=0,0,0,1

; = IMPORTANT =
; = For configuring the rainmeter skin, right-click the visualizer and click on "Open settings".
; = You can also open the variables.ini file located in:
; = "My Documents\Rainmeter\Skins\SysDash\@Resources"

; Small context menu when you right-click the skin
ContextTitle=" Open settings"
ContextAction=[!ActivateConfig "#ROOTCONFIG#\Settings" "general.ini"]
ContextTitle2=" Open variables file"
ContextAction2=["#@#variables.ini"]

[Metadata]
Name=SysDash Dashboard
Author=marcopixel
License=MIT License
Information=An minimalistic, still stylish dashboard-like skin with modular components.

[Variables]
; Includes the variables used for the skin.
@include=#@#variables.ini
@include2=#@#include\MeterStyles.inc

; Measure to get the weather data and properly parse it
[MeasureWeather]
Measure=Plugin
Plugin=WebParser
Url=http://wxdata.weather.com/wxdata/weather/local/TWXX0021?cc=*&unit=m
RegExp=(?siU)<weather ver="(.*)">.*<ut>(.*)</ut>.*<ud>(.*)</ud>.*<us>(.*)</us>.*<up>(.*)</up>.*<ur>(.*)</ur>.*<dnam>(.*)</dnam>.*<lat>(.*)</lat>.*<lon>(.*)</lon>.*<sunr>(.*)</sunr>.*<suns>(.*)</suns>.*<zone>(.*)</zone>.*<tmp>(.*)</tmp>.*<flik>(.*)</flik>.*<t>(.*)</t>.*<icon>(.*)</icon>.*<bar>.*<r>(.*)</r>.*<d>(.*)</d>.*</bar>.*<wind>.*<s>(.*)</s>.*<gust>(.*)</gust>.*<d>(.*)</d>.*<t>(.*)</t>.*</wind>.*<hmid>(.*)</hmid>.*<vis>(.*)</vis>.*<moon>.*<icon>(.*)</icon>.*<t>(.*)</t>.*</moon>

UpdateRate=30
FinishAction=[!Update] [!Update]
[MeasureCurrentCity]
Measure=Plugin
Plugin=WebParser
URL=[MeasureWeather]
StringIndex=7
[MeasureCurrentCode]
Measure=Plugin
Plugin=WebParser
URL=[MeasureWeather]
StringIndex=16
Substitute="":"na"
[MeasureCurrentTemp]
Measure=Plugin
Plugin=WebParser
URL=[MeasureWeather]
StringIndex=13
Substitute="":"N/A"
[MeasureCurrentDesc]
Measure=Plugin
Plugin=WebParser
URL=[MeasureWeather]
StringIndex=15
Substitute="Scattered":""

[MeterWeatherIcon]
Meter=Image
MeasureName=MeasureCurrentCode
Path=#@#images\weather
X=(#Margin#*#Scale#)
Y=(20*#Scale#)
W=(40*#Scale#)
H=(40*#Scale#)
Padding=0,0,0,(20*#Scale#)
ImageTint=#MainColor#
ImageCrop=-30,-30,61,61,5

[MeterWeatherTempText]
Meter=String
MeasureName=MeasureCurrentTemp
MeterStyle=StyleValue
X=(15*#Scale#)R
Y=((80/2)*#Scale#)
Postfix=[\x00B0]

[MeterWeatherCityText]
Meter=String
MeasureName=MeasureCurrentCity
MeterStyle=StyleSecondary
X=((#Width#+#Margin#)*#Scale#)
Y=(30*#Scale#)
InlineSetting4=Color | #FontColor#,255
[MeterWeatherDescText]
Meter=String
MeasureName=MeasureCurrentDesc
MeterStyle=StyleSecondary
X=0r
Y=0R
```

If you want Fahrenheit change 

Url=http://wxdata.weather.com/wxdata/weather/local/TWXX0021?cc=*&unit=m

to

Url=http://wxdata.weather.com/wxdata/weather/local/TWXX0021?cc=*&unit=f

**Comment by marcopixel** [comment]
Will be fixed in the future, check #56 for more information.
